### DISCUSSION TITLE
Feature request: different icon colour for on prem and cloud extension

### DISCUSSION THREAD

**Discussion by luffy3298** [arnoudkooi/SN-Utils#317] [Ideas]
i am using both on-prem and cloud extension. different colour can easily differentiate which is which in my extension bar

![image](https://user-images.githubusercontent.com/7558492/199318470-f47d1eb7-216f-4773-8f25-526073ab9800.png)

**Comment by jacebenson** [comment]
maybe change the color to a .... dark grey?

**Reply by luffy3298** [reply]
any colour is fine. leaving the choice to arnoud 😃

**Reply by luffy3298** [reply]
![image](https://user-images.githubusercontent.com/7558492/199318349-6b1db685-19ba-428d-9c7e-7d7e25bd60cc.png)
This is what i see

**Comment by arnoudkooi** [comment]
Good idea, took the opportunity to do a bit of an icon refresh / improved quality.
Leaving the "legacy color" for OnPrem:
<img width="200" alt="image" src="https://user-images.githubusercontent.com/840781/199626318-74483621-a569-4a1d-98cd-82e96a7f355d.png">
Will be in update 6.2.1.0

**Reply by luffy3298** [reply]
Thank you 🥳
